Makefile

```diff
@@ -29,43 +29,43 @@ $(THRIFT_BUILDDIR)/tests/integration/test.thrift_buildstamp: tests/integration/t
 	rm -f tests/integration/test_thrift/TestService-remote
 	touch $@
 
+.PHONY: venv
+venv:
+	poetry install --all-extras
```

Comment:
you might need `use 3.8` so that people that work on baseplate write 3.8+ code instead of "whatever poetry wants+ code"

Reply:
Hmm... I guess I would also want people to be able to run `poetry env use <whatever>` and then `make .venv` to be able to switch between versions if they need to test with a specific one.

It's also nice to default to the latest version for interactive development (while still having CI ensure you don't break old versions) since it tends to have better tracebacks, improvements to pdb, etc.